src/extension.ts

```diff
@@ -14,11 +14,26 @@ import { ResumeAllSession } from './ResumeAllSession';
 export { ResumeAllSession } from './ResumeAllSession';
 import { SuspendAllSession } from './SuspendAllSession';
 export { SuspendAllSession } from './SuspendAllSession';
+import { ResetDevice } from './ResetDevice';
+export { ResetDevice } from './ResetDevice';
```

Comment:
I see this replicates above command handling. Just curious why it needs to be exported here...